applications/nexus/nex2ecl.cpp

```diff
@@ -16,37 +16,148 @@
 */
 
 #include <cctype>
-
-#include <string>
-
+#include <getopt.h>
 #include <iostream>
+#include <string>
 
 #include <ert/util/util.h>
 
 #include "nexus/util.hpp"
 
+namespace {
+
+static const constexpr char *usage =
+R"(nex2ecl [options]... <nexus_plt_file>
+Try 'nex2ecl --help' for more information.)";
+
+static const constexpr char *help =
+R"(Usage:
+  nex2ecl [options]... <nexus_plt_file>
+  nex2ecl -h | --help
+Options:
+  -f --format                Format the output.
+  -h --help                  Show this screen.
+  -o --output <file>|<path>  Write output to files prefixed by <file> or place
```

Comment:
I think: ` Write output to files with basename <file> or place` will be more familiar to users.

Reply:
Agree